README.md

```diff
@@ -356,16 +356,58 @@ metadata:
   name: example-quayecosystem
 spec:
   quay:
-    routeHost: example-quayecosystem-quay-quay-enterprise.apps.openshift.example.com
+    hostname: example-quayecosystem-quay-quay-enterprise.apps.openshift.example.com
     imagePullSecretName: redhat-pull-secret
 ```
 
+### Methods for Exteral Access
+
+Support is available to access Quay through a number of OpenShift and Kubernetes mechanisms for ingress. When running on OpenShift, a [Route](https://docs.openshift.com/container-platform/4.2/networking/routes/route-configuration.html) is used while a [LoadBalancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) is used. 
+
+The type of external access can be specified by setting the `externalAccessType` using one of the available options in the table below:
+
+| External Access Type | Description |  Notes |
+| --------- | ---------- | ---------- |
+| `Route` | [OpenShift Route](https://docs.openshift.com/container-platform/latest/networking/routes/route-configuration.html) | Can only be specified when running in OpenShift |
+| `LoadBalancer` | [LoadBalancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) | |
+| `NodePort` | [NodePort Service](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) | A dns based hostname or IP address **must** be specified using the `hostname` property of the `quay` resource |
+
+An example of how to specify the `externalAccessType` is shown below
+
+```
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: QuayEcosystem
+metadata:
+  name: example-quayecosystem
+spec:
+  quay:
+    imagePullSecretName: redhat-pull-secret
+    externalAccessType: LoadBalancer
```

Comment:
Will a `LoadBalancer` fail on Openshift? If so, it might be worth mentioning such. If not, then feel free to dismiss this comment :)

Reply:
Nope, will not fail